src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -133,7 +142,7 @@
 int X509_STORE_set_default_paths(X509_STORE *);
 int X509_STORE_set_flags(X509_STORE *, unsigned long);
 void X509_STORE_free(X509_STORE *);
-
+X509_VERIFY_PARAM *_X509_STORE_get0_param(X509_STORE *);
```

Comment:
OpenSSL 1.1.0 has this function (`X509_STORE_get0_param`) and we should mimic the way we implemented versions for < 1.1.0 like https://github.com/pyca/cryptography/blob/master/src/_cffi_src/openssl/rsa.py#L90-L115